src/App.tsx

```diff
@@ -1,8 +1,75 @@
 import React from 'react';
 import './App.css';
+import { Good } from './components/Good';
+import { GoodsList } from './components/GoodsList';
+import { getGoods } from './api/getGoods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+type State = {
+  goods: Good[];
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    goods: [],
+  };
+
+  loadAllGoods = () => {
+    getGoods()
```

Comment:
```suggestion
    getGoods<Good>()
```
это будет более практичный вариант